frontend/src/actions/authActions.js

```diff
@@ -4,14 +4,13 @@ import { SET_USER, LOGOUT_USER } from './types';
 
 export const registerUser = (data, history) => async dispatch => {
     try {
-        const response = await axios.post(process.env.REACT_APP_API_URL + '/api/auth/register', data, {
+        await axios.post(process.env.REACT_APP_API_URL + '/api/auth/register', data, {
```

Comment:
TODO
axios should be configured in one place only, and then just:
await axios.post('/auth/register');

Reply:
you can ask Ewelina how to do it or check other demos

Reply:
done